### DISCUSSION TITLE
Feedback for Proposal

### DISCUSSION THREAD

**Discussion by kbonnici** [International-Slackline-Association/slackmap#4] [General]
I shared some of my thoughts in the Facebook group chat, but I'll try to summarize here so it's properly documented.

### Entity Ownership
I like the concept of `owning` an entity, it gives the user almost a sense of pride that they either discovered or claimed a spot (as a stretch goal, perhaps you could gamify the web app by giving users some kind of rating based on how many lines they have put down on the map?)

The solution for editing line data as a user seems a little bit gatekeeper-y to me, and so I have an idea for a way to potentially circumvent that in the event that a particular user simply refuses to give out perms to edit line data, or is not responding at all.

The proposed solution would be relatively difficult compared to the option I'm about to suggest, and while it's not bulletproof, it might be a better starting point until you flesh out the permissions idea a bit more.

### The Solution
Why not instead give a user the option to `report` a line or entity? This could prompt the user with choices for why they are reporting an entity, and they could perhaps choose from options such as "no longer able to set up due to environmental changes" (maybe the tree got cut down/fell down), or "owner is no longer maintaining the entity", etc. This could trigger some notification on your end that someone has reported a line, so you could determine what to do based on that information, or you could set up an automation task to automatically alter the line data or even delete it depending on the nature of the report.

Not saying this solution is particularly better than the proposed one, but it could be something to think about if you want to minimize dev effort since this is a 100% volunteer project.

**Comment by 0gust1** [comment]
I also find the "request permission before comment/add content" a bit gate-keeper-y (and a bit uneasy / confused by the current vision of rights and permissions).

"Edit" could be restricted. "Comment"/"Add media" authorized but moderated later (and notified to the "owner").
It's like the `report` concept  you propose (but maybe a bit less specific);

To put some context: an aspect to be considered with line/spot "ownership" are also responsibility and access issues.  
In some spots/places (especially for highlines), the spot is "legally" on open-access (public area), but in fact under the responsability of a local group/association that has the trust of the city/state/country officials. It's a kind of problem we're facing with our local group (but also elsewhere, If I trust the community): overenthusiasts/beginners going alone on sensible spots (without connecting with locals), doing things wrong, and finally resulting in spot closing by authorities.

So you still have to put some guards IMHO, **but** don't deter people to contribute content. Not an easy thing to balance (haha).

**Comment by Can-Sahin** [comment]
Let's join the discussion under Rory's one. His reviewing idea is promising and quite aligned with yours.
